### PR TITLE
Update chrome version used on functional tests

### PR DESCRIPTION
**Pull request by PippoRaimondi**
## Description of change

The intent of this PR is to ensure we are running our functional tests in CI with the latest stable chrome version.

## Test instructions

Functional tests pass in circleci

## Checklist

[//]: # "When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/master/docs/Code%20review%20guidelines.md"

- [ ] Has the branch been rebased to master?
- [ ] Automated tests (Any of the following when applicable: Unit, Functional or Acceptance)
- [ ] Manual compatibility testing (Browsers: Chrome, Firefox, IE11, Safari)

### CODE REVIEW

File: test/Dockerfile [hunk 1]
```diff
@@ -52,13 +52,11 @@ RUN apt-get install -y \
 # Install visual test dependencies
 RUN apt-get install -y imagemagick
 
-# Install Chrome (74)
-# We decided to lock on Chrome 74 for now until we solve random error when running functional tests: "Error: There has been an OAuth stateId mismatch"
```
Comment (rafenden) [inline test/Dockerfile]: Looks like we are still getting some random failures.

Comment (PippoRaimondi) [reply test/Dockerfile]: Maybe, will run a couple of times... The error seems to be different than the ones we had previously... fingers crossed

**Comment by PippoRaimondi** [comment]
> Looks fine to me. I'm guessing the failing e2e test will go away after you have rebased?

that's right! and the failing functional might have been a test that needed enhancing, which I did... so it seems to be all good now!

File: test/Dockerfile (hunk 1 above)
Comment (PippoRaimondi) [reply test/Dockerfile]: @rafenden the functional test failing wasn't a flaky one, instead one built in a way that passed in previous versions but failed on newer ones, I enhanced it and it passed... so this is 5-6 runs proven to be solid... I will rerun it once more and if it goes green again I will merge! worst case scenario I can revert!

**Pull request merged**